src/types/url.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongUrl = StrongType<string>;
 
-export function makeUrl(fallback: string, initial?: string | null): StrongUrl {
+export function makeUrl(fallback: string, initial: string | null | undefined): StrongUrl {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.